builds/checkin/dotnet.yaml

```diff
@@ -67,3 +67,44 @@ jobs:
           testRunner: VSTest
           testResultsFiles: '**/TestResults/*.trx'
         condition: succeededOrFailed()
+
+################################################################################
+  - job: code_coverage
+################################################################################
+    displayName: Code coverage
+    variables:
+      coverage.goal: 64
```

Comment:
suggest to define it in pipeline variable instead.

Reply:
The iotedged code coverage goal is a job variable; I'm following the existing pattern. Why make it a pipeline variable if only one job needs it?

Reply:
so you can change it by editing the pipeline (not query time) instead of yaml change.

Reply:
Ah, I see what you're saying. But I think there's value in storing coverage goals in source control, alongside the code. That way if you send an older commit through our build system, the goal value will match the state of our code at the time of that commit.

If you publish a PR that increases coverage, it's easy enough to bump the goal in that PR as well.